### PR TITLE
add make draft_release target

### PR DESCRIPTION
**Pull request by dsvoronin**
produces something like this:

image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,6 +67,21 @@ publish_to_maven_local:
...
+# precondition:
+# - installed CLI: https://cli.github.com/
+# - push $(version) branch
+#
+# post actions:
+# - go to link in output
+# - edit notes
+# - publish release
+draft_release:
+	gh release create $(version) \
```
Comment (eugene-krivobokov) [inline Makefile:96]: I'd like to have more fail-fast checks for preconditions.
At least for variables.

```bash
@[ "${version}" ] || ( echo ">> version is not set"; exit 1 )
```
https://stackoverflow.com/questions/10858261/abort-makefile-if-variable-not-set

File: Makefile [hunk 2]
```diff
@@ -67,6 +67,21 @@ publish_to_maven_local:
...
 
+# precondition:
+# - installed CLI: https://cli.github.com/
+# - push $(version) branch
+#
+# post actions:
+# - go to link in output
+# - edit notes
+# - publish release
+draft_release:
```
Comment (eugene-krivobokov) [inline Makefile:93]: Please, mention this in https://avito-tech.github.io/avito-android/docs/release/

File: Makefile (hunk 1 above)
Comment (dsvoronin) [reply Makefile:96]: Thanks, didn't know how to properly add it; let's fix

File: Makefile (hunk 2 above)
Comment (dsvoronin) [reply Makefile:93]: done

File: Makefile (hunk 1 above)
Comment (dsvoronin) [reply Makefile:96]: done for vars

File: Makefile [hunk 3]
```diff
@@ -49,6 +49,21 @@ ifdef stacktrace
...
+# Params:
+#   1. Variable name(s) to test.
+#   2. (optional) Error message to print.
+check_defined = \
+    $(strip $(foreach 1,$1, \
+        $(call __check_defined,$1,$(strip $(value 2)))))
+__check_defined = \
+    $(if $(value $1),, \
+      $(error Undefined $1$(if $2, ($2))))
+
```
Comment (eugene-krivobokov) [inline Makefile:66]: Oh My Bash

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**